guides/common/modules/con_permissions-for-remote-execution.adoc

```diff
@@ -26,12 +31,12 @@ name = Reboot and host.name ~ *.staging.example.com
 name = "Restart service" and host_group.name = webservers
 ----
 
-The first line in this example permits the user to apply the *Reboot* template to one selected host.
+The first line in this example permits you to apply the *Reboot* template to one selected host.
```

Comment:
Anthropomorphism. The software should never be the one restricting "you". Check out the IBM Style guide for a detailed explanation.